app/templates/base/home.html

```diff
@@ -5,16 +5,133 @@
 
 {% block main_column %}
 
+  <ul class="tabs">
```

Comment:
So essentially this is an aggregations view of pretty much all the other views.
What does it mean for existing views and redirects?

Reply:
Yep, this is a fair point. The user's homepage aggregates their apps, their data, their tools etc. The only one of these that has an equivalent page at the moment is tools, since the list pages for the others are superuser "show all" views.

Possibly what we could do is if a non-superuser manually enters `/apps` we could redirect them to `/#Webapps` in order to show just their apps, etc.